src/components/NewProduct/NewProduct.js

```diff
@@ -0,0 +1,56 @@
+import React, { useState, useEffect } from 'react';
+import ProductForm from '../ProductForm/ProductForm';
+import './NewProduct.scss';
+const NewProduct = () => {
+  const [newPhoto, setNewPhoto] = useState([]);
+  const [count, setCount] = useState(0);
+
+  useEffect(() => {
+    fetch('./data/New.json', {
+      method: 'GET',
+    })
+      .then(res => res.json())
+      .then(data => [setNewPhoto(data)]);
+  }, []);
```

Comment:
```suggestion
  useEffect(() => {
    fetch('./data/New.json')
      .then(res => res.json())
      .then(data => setNewPhoto(data));
  }, []);
```
GET 메서드를 사용할 때는 위와 같이 축약할 수 있습니다.
혹시 이전 코드에서 setNewPhoto가 [] 감싸져 있었는데 에러가 없었을까요?

Reply:
감사합니다 도현님 에러는 없었는데 수정 했습니다.